.circleci/config.yml

```diff
@@ -1,11 +1,11 @@
 anchors:
-  - &latest-xcode "12.5.1"
-  - &latest-ios   "14.5"
-  - &min-ios      "13.7"
-  - &chruby       "2.7.3"
-  - &beta-xcode   "13.0.0"
+  - &latest-xcode "13.2.1"
+  - &latest-ios   "15.0"
```

Comment:
If you are bumping Xcode to `13.2.1` you need to bump latest iOS to `15.2`, min iOS to `14.5` and chruby any of the available versions: `2.6.9, 2.7.5, 3.0.3` as per the [what is available in the xcode 13.2.1 container](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v6690/index.html).  (link to this [in the circleci anncounment post](https://discuss.circleci.com/t/xcode-13-2-1-released/42334)).